### PR TITLE
Add dos.x and projwfc.x code config to computers

### PR DESCRIPTION
**Pull request by unkcpz**
- Correct file name in eiger code config of projwfc.
- Add dos.x and projwfc.x to imxgesrv and daint.

### CODE REVIEW

File: imxgesrv1.epfl.ch/codes/dos-6.7.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+label: "dos-6.7"
+description: "QuantumESPRESSO 6.7.0 dos.x code"
+input_plugin: "quantumespresso.dos"
+on_computer: true
+remote_abs_path: "/home/software/apps/codes/intel/19.1.3.304/quantum-espresso/6.7.0_MPI_MKL/bin/dos.x"
+computer: "imxgesrv1"
```
Comment (yakutovicha) [inline imxgesrv1.epfl.ch/codes/dos-6.7.yaml:7]:
```suggestion
```

**Review by ltalirz — APPROVED** [review]
I can't comment on the correctness of this PR.

If you would like to be able to add new codes without review, let me know and I can add you as an admin.

File: imxgesrv1.epfl.ch/codes/projwfc-6.7.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+label: "projwfc-6.7"
+description: "QuantumESPRESSO 6.7.0 projwfc.x code"
+input_plugin: "quantumespresso.projwfc"
+on_computer: true
+remote_abs_path: "/home/software/apps/codes/intel/19.1.3.304/quantum-espresso/6.7.0_MPI_MKL/bin/projwfc.x"
+computer: "imxgesrv1"
```
Comment (yakutovicha) [inline imxgesrv1.epfl.ch/codes/projwfc-6.7.yaml:7]:
```suggestion
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
A small change request, otherwise good to go.

File: imxgesrv1.epfl.ch/codes/dos-6.7.yaml (hunk 1 above)
Comment (unkcpz) [reply imxgesrv1.epfl.ch/codes/dos-6.7.yaml]: Thanks! I also remove the line from `pw.yaml` and `ph.yaml`.

**Comment by unkcpz** [comment]
> If you would like to be able to add new codes without review, let me know and I can add you as an admin.

Thanks, @ltalirz. Please add me to admins. I'll try my best to have every PR approved before merging, better to be.

**Comment by csadorf** [comment]
Since this PR has already been reviewed by @ltalirz and @yakutovicha , I'll remove this from my queue.

**Comment by unkcpz** [comment]
Merged. Thanks!

**Pull request merged**